README.md

```diff
@@ -0,0 +1,55 @@
+# Netflix AutoSkip
+
+## What is this?
+This is a simple plugin for Chromium based and Firefox based browsers that skips the following Netflix screens:
+
+- The "Who's watching?" screen
+- The "Skip Intro" button
+- The "Continue Watching" button
```

Comment:
this extension doesn't skip the "who's watching" screen, but it auto clicks "next episode"
```suggestion
This is a simple plugin for Chromium based and Firefox based browsers that skips the following Netflix screens:

- The "Skip Intro" button
- The "Continue Watching" button
- The "Next Episode" button (credits)
```